app/serializers/group_user_with_custom_fields_serializer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class GroupUserWithCustomFieldsSerializer < UserWithCustomFieldsSerializer
+  include UserPrimaryGroupMixin
+
+  attributes :name, :title, :last_posted_at, :last_seen_at, :added_at, :timezone, :status
+
+  def timezone
+    user.user_option.timezone
+  end
+
+  def include_added_at?
+    object.respond_to? :added_at
+  end
```

Comment:
I know `GroupUserSerializer` has that but… is it ever true?

Reply:
Yes